src/components/index/DeFiChainEcoSystemSection.tsx

```diff
@@ -116,27 +116,6 @@ export function DeFiChainEcoSystemSection(): JSX.Element {
           />
         </div>
       </Container>
-      <div
-        className={classNames(
-          "absolute h-1/3 w-1/3 lg:bottom-[200px] md:bottom-[100px] left-0 bg-contain bg-no-repeat mix-blend-screen bg-[url('/assets/img/ecosystem/dfi-logo-vector.png')]",
```

Comment:
do we need to remove these files too?

Reply:
If im not wrong, the designers want it to be part of the background, that's why i removed it